src/backend/main.ts

```diff
@@ -63,9 +63,8 @@ const initialize = async () => {
       developmentServer: process.env.NODE_ENV === "development",
       authConfig: {
         clientId,
-        scope: "openid email profile organization imodelhub context-registry-service:read-only product-settings-service urlps-third-party offline_access",

```

Comment:
without offline_access we won't get a refresh token, right?

Reply:
I copied the list of scopes from developer.bentley.com. offline_access isn't in the list. I guess native clients get `offline_access` by default? I'll follow up with that team, but I'll add this scope back.

Reply:
Ah okay, maybe developer.bentley.com adds it automatically or it's not needed anymore for a client created there.